### ISSUE TITLE
Resolve all eslint errors

### ISSUE THREAD

**Issue by LoTerence** [hackforla/CivicTechJobs#609]
### Overview

As a developer, we should resolve all eslint errors, so that the code maintains a high standard of quality and follows best practices, improving readability, consistency, and reducing the risk of bugs.

If you go into the frontend folder, and run `npm run lint`, 47 eslint errors and warnings come up.

Screenshot:
![image](https://github.com/user-attachments/assets/fc16c925-f12b-44dc-a765-2775ac26fabc)

### Action Items

- [x] resolve all eslint errors

### Resources/Instructions

[Resources](https://hackforla.github.io/CivicTechJobs/resources/)

**Comment by nooriaali9** [comment]
Hi @LoTerence this issue looks good, if it is currently being worked on, it can be moved to In progress.

**Comment by LoTerence** [comment]
@nooriaali9  I moved it to prioritized backlog.

Hey @irais-valenzuela if you are still working on this issue, could you move it to "In Progress" please? This is for PM stuff

**Comment by irais-valenzuela** [comment]
Hi @LoTerence, Thank you for writing up this ticket! I wanted to let you know that I'll be unassigning myself to avoid holding things up on your end. I have a few more interviews coming up, and I won’t have much time to dedicate to this.

**Comment by nooriaali9** [comment]
Hi @LoTerence I've unassigned Ida from this ticket, is there someone else you'd like to assign this to? or do we move it back to prioritized backlog?

**Comment by nooriaali9** [comment]
@irais-valenzuela Please update on this ticket - with the progress you've made on this!

**Comment by irais-valenzuela** [comment]
This ticket has been completed. I'm just waiting on code review for it to be merged. The PR is [here](https://github.com/hackforla/CivicTechJobs/pull/639)

**Comment by LoTerence** [comment]
PR merged, thank you Ida!

There are still 16 eslint warnings left. I will make a new PR to clean up the rest.

**Comment by LoTerence** [comment]
Resolved with PR #640
